pkg/apis/sparkoperator.k8s.io/v1beta2/types.go

```diff
@@ -162,6 +162,16 @@ type ScheduledSparkApplicationList struct {
 	Items           []ScheduledSparkApplication `json:"items,omitempty"`
 }
 
+// Specific SparkUI config parameters
+type UIConfig struct {
+	// Ingress class used for k8s annotation: kubernetes.io/ingressclass
+	IngressClass *string `json:"ingressClass,omitempty"`
```

Comment:
Golang comment for fields should start with the name of the field, like `IngressClass defines the value for annotation ...`. Ditto for the fields below. Also please add `// +optional` for the fields since they all look optional.

Reply:
i adapted the comments. Thanks for your help.